vignettes/paper.Rmd

```diff
@@ -174,7 +174,8 @@ Finally, in Section 4, we discuss limitations of the approach and possible direc
 
 The aim of the ClockBoard zoning system is to tackle the issues associated with available zoning systems and to provide a standard template for analysis of the geographic distribution of urban phenomena between cities internationally.
 As outlined above, there is a need for open access zoning systems that are readily available and consistent.
-The requirements of urban analysts, geographers, transport modellers and other working with geographic data across cities are diverse, however, motivating our approach which is free, open, reproducible and easy to extend: modifications to the ClockBoard system and completely new zoning systems could build the approach.
```

Comment:
See comments below, I'm not sure at all what you're trying to say here

Reply:
Yes it's not a very clear couple of sentences, good point. Essentially yes, our aim is for ClockBoard to be useful for many people. Over the last few days I've thought it useful to refer to at least 3 levels of abstraction: zoning systems being the highest level, segmented annuli being a type of zoning system, and ClockBoard being an implementation of the segmented annuli with specific parameters. Possibly not worth saying that here but worth being clear and I think the essence of what I'm trying to say can be communicated with fewer words.